docs/source/tutorial_quick.rst

```diff
@@ -57,26 +59,89 @@ Let's try asking the model a question ourselves.
 .. code-block:: bash
 
   # interact with saved model
-  python examples/interactive.py -mf /tmp/babi_memnn
+  python examples/interactive.py -mf /tmp/babi_memnn -ecands vocab
   ...
   Enter your message: John went to the hallway.\n Where is John?
 
 Hopefully the model gets this right!
 
 
-Simple model
-------------
+
+Train a Transformer on Twitter
+------------------------------
+
+Now let's try training a Transformer (Vaswani, et al 2017) ranker model.
+Make sure to complete this section on a GPU with PyTorch installed.
+
+We'll be training on the Twitter task, which is a dataset of tweets and replies.
+There's more information on tasks in these docs,
+including a full list of `tasks <http://parl.ai/docs/tasks.html>`_ and
+`instructions <http://parl.ai/docs/tutorial_basic.html#training-and-evaluating-existing-agents>`_
+on specifying arguments for training and evaluation (like the ``-t <task>`` argument used here).
+
+Let's begin again by printing the first few examples.
+
+.. code-block:: bash
+
+  # display first examples from twitter dataset
+  python examples/display_data.py -t twitter
+
+Now, we'll train the model. This will take a while to reach convergence.
+
+.. code-block:: bash
+
+  # train transformer ranker
+  python examples/train_model.py -t twitter -mf /tmp/tr_twitter -m transformer/ranker -bs 10 -vtim 3600 -cands batch -ecands batch --data-parallel True
+
+You can modify some of the command line arguments we use here -
+we set batch size to 10, run validation every 3600 seconds,
+and take candidates from the batch for training and evaluation.
+
+The train model script will by default save the model after achieving best validation results so far.
+The Twitter task is quite large, and validation is run by default after each epoch (full pass through the train data),
+but we want to save our model more frequently so we set validation to run once an hour with ``-vtim 3600``.
+
+This train model script evaluates the model on the valid and test sets at the end of training, but if we wanted to evaluate a saved model -
+perhaps to compare the results of our newly trained Transformer against a pretrained seq2seq baseline from our `Model Zoo <http://parl.ai/docs/zoo.html>`_,
+we could do the following:
+
+.. code-block:: bash
+
+  # Evaluate seq2seq model trained on twitter from our model zoo
+  python examples/eval_model.py -t twitter -m legacy:seq2seq:0 -mf models:twitter/seq2seq/twitter_seq2seq_model
+
+
+Let's print some of the transformer's predictions with the same display_model script from above.
+
+.. code-block:: bash
+
+  # display predictions for model saved at specific file on twitter
+  python examples/display_model.py -t twitter -mf /tmp/tr_twitter -ecands batch
+
+And finally, we can talk to the model interactively as before:
+
+.. code-block:: bash
+
+  # interact with saved model
+  python examples/interactive.py -mf /tmp/tr_twitter
```

Comment:
This looks good! Note here, this command won't work. We'll probably have to specify a set of fixed candidates for the file to rank at interactive time (i.e. `-ecands fixed -fcp <some txt file with candidates to rank`)... although some have suggested that we should make the generation of this file automatic if we specify the task, so may change in the future

Reply:
Yes! I had meant to ask about this and completely forgot - since we use -ecands vocab for babi but we can't do that here. Hmm it would be nice if we made a file for this part of the task or provided a script to generate this file for a particular task.

Reply:
ok I'm going to remove this code block for now since it's not very important and I don't want to explain -ecands or -cands in the quick start